career/360-review.md

```diff
@@ -0,0 +1,35 @@
+# 360 Reviews
+
+ A 360 review is a tool for gathering feedback for an individual (evaluee). It is done by interviewing people who work with the evaluee in different contexts or with different roles.
+
+## Goal
+
+The goal of the 360 review is to provide in-depth feedback for the evaluee so that they can reflect on how others perceive their work. The ultimate goal is growth, and it is achieved by reflection. It is usually the most extensive feedback that the evaluee receives in the year.
+
+
+## Preparation
+
+The preparation consists of getting to know what is the evaluee missing the most. What do they really want to know?
+Set up a short interview with enough anticipation. 1 or 2 weeks in advance. Let clear the intentions in the agenda. Make them prepare a list of questions. During the interview, dig a bit deeper into the list. Why do they want to know that? Whom do they want to hear from? The list may include a close client and their manager. Is there anything that's not on the list but may be interesting? Use your experience from other 360 reviews to suggest what they may want to know.
+
+
+## Conducting the interviews
+
+Once we have the list, set up interviews with the people who work close to the evaluee. Let them know 1 or 2 weeks in advance so that they can prepare. Let them know some of the questions that you'll be asking so that they can prepare mentally. Once the invite is sent, they'll pay much more attention.
+
+There is abundant material on the internet about how to conduct a good interview. Go and read it. The short version is that you must listen. Listen and go beyond the prepared questions with follow up questions. Dig into juicy details. If pairing with them is "nice", ask specifically what it is "nice" about pairing with them.
+
+Try to balance a good mix of positive and constructive feedback. Believe me; it is hard to get constructive feedback. The evaluee will appreciate enormously the effort that you are putting into getting to the bottom of the question. How can they keep improving?
+
+## Presenting the results
+
+The results may be overwhelming to the evaluee. People are not used to receiving such amounts of feedback in a single session. Therefore, it is crucial to present it in a way that helps the absorption. A written format can be used together with an in-person meeting explaining the results. However, be extra cautious with the written form when giving feedback. It is easy to misinterpret.
```

Comment:
We should maybe note here that the person being evaluated can be offered to receive their feedback in written form before the meeting so they have a bit of time to prepare questions and digest it before the actual 360 review meeting with their direct manager.

Reply:
As long as we all understand the limitations of the written format, I'm alright with that. The EM will have enough freedom to do it as they please.